nimble/host/src/ble_att_svr.c

```diff
@@ -171,7 +171,8 @@ ble_att_svr_find_by_handle(uint16_t handle_id)
 /**
  * Find a host attribute by UUID.
  *
- * @param uuid                  The ble_uuid_t to search for
+ * @param uuid                  The ble_uuid_t to search for; null means
+ *                                  find any attribute.
```

Comment:
looks like not any, but fist from attribute list or next after prev

Reply:
I meant any type of attribute. The behavior you describe is not related to the `uuid` param. It is described under the `prev` param.